tools/src/main/java/com/nedap/archie/rules/evaluation/evaluators/BinaryOperatorEvaluator.java

```diff
@@ -348,6 +349,26 @@ private ValueList evaluateRelOpOperator(RuleEvaluation<?> evaluation, BinaryOper
             //want otherwise? Use for_all/every
             result.addValue(stringOperandEvaluator.evaluateMultipleValuesStringRelOp(statement, leftValues, rightValues));
 
+            return result;
+        } else if (PrimitiveType.Date.equals(leftValues.getType()) || PrimitiveType.Date.equals(rightValues.getType()) ||
```

Comment:
Maybe create two EnumSets and call a contains on them to improve readability? :) Could even be statically created for a (very) minor performance increa.